src/Passport.php

```diff
@@ -651,4 +658,25 @@ public static function withoutCookieSerialization()
 
         return new static;
     }
+
+    /**
+     * Determine if refresh tokens must be revoked.
+     *
+     * @return bool
+     */
+    public static function revokeRefreshTokens()
+    {
+        return static::$revokeRefreshTokens;
+    }
```

Comment:
Like I said in the previous PR, you can remove this method. The property can be called directly.